content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -111,6 +111,9 @@ sudo yum list --showduplicates kubeadm --disableexcludes=kubernetes
 ```
 
 {{% /tab %}}
+
+If you don't see the version you expect to upgrade to, [verify if the Kubernetes package repositories are used.](/docs/tasks/administer-cluster/kubeadm/change-package-repository/#verifying-if-the-kubernetes-package-repositories-are-used)
+
 {{< /tabs >}}
```

Comment:
should be after this line

or otherwise it doesn't render if it's between the tag and tabs tags.

https://deploy-preview-48608--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/

Reply:
Apologies, corrected now - hope it's fine :)